apps/api/migrations/novu-integrations/novu-integrations.migration.ts

```diff
@@ -42,6 +44,7 @@ const createNovuIntegration = async (
     _organizationId: environment._organizationId,
     providerId,
     channel,
+    identifier: `${slugify(providerId, { lower: true, strict: true })}-${shortid.generate()}`,
```

Comment:
Good catch! is it the same way we do it on the client and api?

Reply:
Not exactly we use the provider name that we have hard coded on the frontend maybe it should be here as well?

Reply:
I added it :)